src/wms/src/test/java/org/geoserver/wms/capabilities/GetCapabilitiesReponseTest.java

```diff
@@ -179,4 +180,31 @@ public void testInternationalContentAnyLanguage() throws Exception {
 
         assertXpathEvaluatesTo("parola chiave", natureGroup + "/KeywordList/Keyword", result);
     }
+
+    @Test
+    public void testAcceptLanguagesParameter() throws Exception {
+        Catalog catalog = getCatalog();
```

Comment:
Same as above

Reply:
Catalog and FeatureTypeInfo are needed to have language links to appear.